packages/cli/src/prebuild.ts

```diff
@@ -591,12 +591,28 @@ export const prebuild = async (options: {
       ),
     });
 
+    const favIconAsset = imageAssets.find(
+      (asset) => asset.id === siteData.build.pages.meta?.faviconAssetId
+    );
     const pageExports = `/* eslint-disable */
 /* This is a auto generated file for building the project */ \n
 
 import { Fragment, useState } from "react";
+import type { FontAsset, ImageAsset } from "@webstudio-is/sdk";
 import { useResource } from "@webstudio-is/react-sdk";
 ${componentImports}
+export const favIconAsset: ImageAsset | undefined = ${
+      favIconAsset ? JSON.stringify(favIconAsset) : "undefined"
```

Comment:
`JSON.stringify(undefined) === undefined`

Reply:
having undefined as a string in the database would be definitely weird